TransTest.hs

```diff
@@ -0,0 +1,131 @@
+import System.Exit (exitFailure)
+
+import Data.List
+
+import Tuura.Concept.STG
+import Tuura.Concept.STG.Translation
+
+main :: IO ()
+main = do
+    testDotGOutput
+    testHandshakeConcept
+    testOrGateConcept
+    testTwoAndGates
+    testTwoDifferentCElements
+
+testDotGOutput :: IO ()
+testDotGOutput = do
+    putStrLn "=== testDotGOutput"
+    assertEq threeSignalsConcept expected
+  where
+    expected = ".model out\n"          ++
+               ".inputs A\n"           ++
+               ".outputs B\n"          ++
+               ".internal C\n"         ++
+               ".graph\n"              ++
+               "A0 A+\n"               ++
+               "A+ A1\n"               ++
+               "A1 A-\n"               ++
+               "A- A0\n"               ++
+               "B0 B+\n"               ++
+               "B+ B1\n"               ++
+               "B1 B-\n"               ++
+               "B- B0\n"               ++
+               "C0 C+\n"               ++
+               "C+ C1\n"               ++
+               "C1 C-\n"               ++
+               "C- C0\n"               ++
+               ".marking {A0 B1 C0}\n" ++
+               ".end\n"
+
+testHandshakeConcept :: IO ()
+testHandshakeConcept = do
+    putStrLn "===testHandshakeConcept"
+    assertEq (sort handshakeConcept) (sort expected)
+  where
```

Comment:
Let's put the definition of `handshakeConcept` right here. I see no reason to keep it in a separate function -- it just makes it harder to understand what is being tested.

Same for other concepts that we are testing.